README.md

```diff
@@ -11,10 +11,13 @@
 - HTML
 - CSS
 - JavaScript
+- Tailwind
 
 ## Live Demo :arrow_forward:
 
```

Comment:
- [ ] `required`
- [ ] Please add a live demo as the requirements state. 🥂

Reply:
@mwanawabangona I am working on a feature branch that will have to merge for the default branch to be deployable

Reply:
@Kelvin-Ben you can deploy the feature branch am trying to reach you on slack

Reply:
okay

Reply:
@danielufeli can you guide me please